src/utils/markdown/twoslash-tabindex/twoslash-tabindex-script.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Scrollable code-container blocks should have tabindex="0" for keyboard accessibility.
+ *
+ * These are added in rehype-transform, but can be removed with JS on elements that
+ * cannot be horizontally scrolled.
+ *
+ * https://github.com/unicorn-utterances/unicorn-utterances/issues/738
+ */
+export const enableTwoslashTabindex = () => {
```

Comment:
Non-blocking: Nice! Works great when JavaScript is enabled. With it disabled, it seems like Chrome is still working around this so it behaves well, but if `tabindex="0"` was added by default, and then removed/added on load and resize, we could expand the no-JS support.

Not a crucial part of the change in my opinion.

Reply:
This was meant to be working! I thought my rehype code was doing this, but apparently not - thanks for catching that!